pkg/modules/libreoffice/routes.go

```diff
@@ -81,6 +83,7 @@ func convertRoute(libreOffice libreofficeapi.Uno, engine gotenberg.PdfEngine) ap
 					SinglePageSheets:         singlePageSheets,
 					LosslessImageCompression: losslessImageCompression,
 					ReduceImageResolution:    reduceImageResolution,
+					ExportNotesInMargin:      exportNotesInMargin,
```

Comment:
Could you also move this before `LosslessImageCompression`?

Reply:
Done